### PR TITLE
Update Set-PnPTenantCdnEnabled.md

### PR DESCRIPTION
**Pull request by sparkitect**
Added reference link
(this is my first time contributing)

### CODE REVIEW

**Comment by get-itips** [comment]
@sparkitect Jim,
Thanks for your contribution, we appreciate it, PnP cmdlet documentation is directly reflected from source code on other repo, if we change anything here, it will then get overwrote by that process later.
So the change must be done on this other repo https://github.com/pnp/PnP-PowerShell/ I do not know how they add reference links to the markdown creation process, probably @erwinvanhunen knows better.

For example, if you see here https://github.com/pnp/PnP-PowerShell/blob/dev/Commands/Admin/SetTenantCdnEnabled.cs you will see the source code of the cmdlet and that includes the documentation of it.

Will try to help to get your contribution added.

Thank you

File: sharepoint/sharepoint-ps/sharepoint-pnp/Set-PnPTenantCdnEnabled.md [hunk 1]
```diff
@@ -76,4 +76,6 @@ Accept pipeline input: False
 
 ## RELATED LINKS
 
-[SharePoint Developer Patterns and Practices](https://aka.ms/sppnp)
\ No newline at end of file
+[SharePoint Developer Patterns and Practices](https://aka.ms/sppnp)
+
+[Content Delivery Networks (CDNs)](https://docs.microsoft.com/en-us/office365/enterprise/content-delivery-networks)
```
Comment (tseward) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Set-PnPTenantCdnEnabled.md:81]: Need to remove `en-us` from the link.

**Comment by erwinvanhunen** [comment]
@get-itips is correct: documentation changes are to be made directly on the source code of the actual cmdlet: https://github.com/pnp/PnP-PowerShell/blob/dev/Commands/Admin/SetTenantCdnEnabled.cs. The PnP PowerShell documentation is generated based on the source code, which will us to retain the documentation close to the actual code that is being executed. We also automatically generate the built-in help for PnP PowerShell using the source code.

**Comment by get-itips** [comment]
> @get-itips is correct: documentation changes are to be made directly on the source code of the actual cmdlet: https://github.com/pnp/PnP-PowerShell/blob/dev/Commands/Admin/SetTenantCdnEnabled.cs. The PnP PowerShell documentation is generated based on the source code, which will us to retain the documentation close to the actual code that is being executed. We also automatically generate the built-in help for PnP PowerShell using the source code.

Thanks @erwinvanhunen Erwin for your comment.
@sparkitect Jim, I am sorry, however, we will need to close this one, your contribution will need to be handled on the repo Erwin and I referenced, Thank you.

**Pull request closed without merging**